internal/broker/broker.go

```diff
@@ -97,7 +98,15 @@ func GetMessages(b *AMQPBroker, queue string) <-chan amqp.Delivery {
 }
 
 // SendMessage sends message to RabbitMQ if the upload is finished
-func SendMessage(b *AMQPBroker, corrID, exchange, routingKey string , body []byte) error {
+func SendMessage(b *AMQPBroker, corrID, exchange, routingKey string, body []byte) error {
+	// Set channel
+	if e := b.Channel.Confirm(false); e != nil {
```

Comment:
What is this? What is it needed for?

Reply:
`Confirm puts this channel into confirm mode so that the client can ensure all publishings have successfully been received by the server.`

Reply:
Does this block?

Reply:
> Does this block?

block what? be more specific.

Reply:
I think Jon means if it's blocking call or async one. And my guess would be that it's blocking, but I don't think that it matters.

Reply:
If this is written to a channel, my guess is async. But I am not sure... The python code made blocking amqp calls.

Reply:
It's blocking in terms of programming, not in terms of messaging. I mean, it blocks for a time needed to put a message to a broker, but no longer.